src/internalUtils/components/ExampleList/VariableDetails.tsx

```
@@ -0,0 +1,38 @@
+
+  return (
+    <div style={{ width: calculatedWidth, display: "flex" }}>
+      <CopyableCode id="example-list-item-variable" value={cssVariable} style={{ marginRight: "auto" }} />
+
+      {values && (
+        <div id="example-list-item-values" style={{ margin: "unset", whiteSpace: "nowrap", paddingLeft: "1rem" }}>
+          (
+          {values.map((value, index) => {
+            return (
+              <>
+                <span>{value}</span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)